e2e/mocha.test.js

```diff
@@ -13,6 +13,22 @@ describe('examples/mocha', () => {
     BUILDKITE_ANALYTICS_DEBUG_ENABLED: "true"
   }
 
+  describe('when token is defined through reporter options', () => {
+    test('it uses the correct token', (done) => {
+      exec('mocha --reporter mocha-multi-reporters --reporter-options configFile=token-config.json',
+      { cwd, env: { ...env, BUILDKITE_ANALYTICS_TOKEN: undefined } }, (error, stdout, stderr) => {
+        expect(stdout).toMatch(/.*Test Analytics Sending: ({.*})/m);
+
+        const jsonMatch = stdout.match(/.*Test Analytics Sending: ({.*})/m)
+        const json = JSON.parse(jsonMatch[1])["headers"]
+
+        expect(json).toHaveProperty("Authorization", 'Token token="abc"')
```

Comment:
Do we have similar tests to verify the `BUILDKITE_ANALYTICS_TOKEN` method still works?

Reply:
Sorry, forgot to add those thanks for the reminder 👍